src/components/fetchhelper.ts

```diff
@@ -1,12 +1,12 @@
-export function getFetchPromise(request) {
-    var headers = request.headers || {};
+export function getFetchPromise(request: any): Promise<any> {
```

Comment:
```suggestion
export function getFetchPromise(request: never): Promise<never> {
```

If it's 100% impossible to determine exactly what kind of objects we will get here, use ``never`` or ``unknown``. As more things we have without strictly typing, more bugs can slip.

Reply:
These return a fetch response. I see tutorials using templating for the response type, for the arguments I suspect there is something ready to use.

Reply:
Fixed half of it